alerting/src/main/kotlin/org/opensearch/alerting/DocumentReturningMonitorRunner.kt

```diff
@@ -121,18 +146,41 @@ object DocumentReturningMonitorRunner : MonitorRunner {
 
         val index = (monitor.inputs[0] as DocLevelMonitorInput).indices[0]
 
+        // TODO: modify findings such that there is a finding per document
+        val findings = mutableListOf<String>()
         queryToDocIds.forEach {
             val queryTriggeredDocs = it.value.intersect(triggerResult.triggeredDocs)
             // TODO: Update finding only if it is not dry run, else return the findings
             if (queryTriggeredDocs.isNotEmpty() && !dryrun && monitor.id != Monitor.NO_ID) {
                 val findingId = createFindings(monitor, monitorCtx, index, it.key, queryTriggeredDocs, trigger)
+                findings.add(findingId)
                 // TODO: check if need to create alert, if so create it and point it to FindingId
                 // TODO: run action as well, but this mat need to be throttled based on Mo's comment for bucket level alerting
```

Comment:
address this now since we are adding actions here. Open to have ToDo for throttling.

Reply:
done